packages/cms-cli/package.json

```diff
@@ -24,7 +24,7 @@
   },
   "bin": {
     "banjo": "./bin/cli.js",
```

Comment:
I think we should go ahead and remove this now.

Reply:
Awe, I was going to keep it in as an easter egg

Reply:
The drawback with doing that is that it sort of pollutes the set of commands installed when a user installs globally, which is what we recommend.